src/Features/Core/Portable/EditAndContinue/BidirectionalMap.cs

```diff
@@ -31,11 +31,11 @@ public BidirectionalMap(IEnumerable<KeyValuePair<T, T>> entries)
             Reverse = reverseMap;
         }
 
-        public static BidirectionalMap<T> FromMatch(Match<T> match)
+        public static BidirectionalMap<SyntaxNode> FromMatch(Match<SyntaxNode> match)
```

Comment:
The public API also needs to validate the input nodes, see
http://source.roslyn.io/#Microsoft.CodeAnalysis.CSharp.Features/EditAndContinue/SyntaxUtilities.cs